subsys/suit/ipuc/src/suit_ipuc_sdfw.c

```diff
@@ -371,6 +362,21 @@ suit_plat_err_t suit_ipuc_sdfw_write(int ipc_client_id, struct zcbor_string *com
 			return SUIT_PLAT_ERR_IO;
 		}
 
+		if ((ipuc_sink.write != NULL) && (offset > ipuc_entry->write_peek_offset)) {
```

Comment:
Please add the `ipuc_sink.seek == NULL` to the condition too.

Reply:
You mean `ipuc_sink.seek != NULL`? Ok, added